doc/admin-deployment.md

```diff
@@ -63,6 +63,10 @@ This policy also prevents the new user welcome page from launching upon Privacy
 }
 ```
 
+### Chrome on Mac
+
+Follow instructions [here](https://www.chromium.org/administrators/configuring-policy-for-extensions/#mac) to add an extension policy via plist file. Depending on your shell environment, you may need to change the syntax of some commands. Use [this plist file](/doc/configuration.plist) as a template, subtituting the extension ID for the ID of your locally installed Privacy Badger. Add keys in the format of `showCounter` to apply additional settings.
```

Comment:
>Depending on your shell environment, you may need to change the syntax of some commands.

Should probably explain this with an example. Or remove entirely.

> Add keys in the format of `showCounter` to apply additional settings.

Can probably remove this line, or replace it with something like "The following policy disables the tracker count badge on Privacy Badger's icon."

Reply:
Should we move all the policy examples into a subdirectory? Maybe `doc/sample-admin-policies`?

Reply:
Good call, I removed the unnecessary instructions and moved the 3 policy examples into a new folder